packages/eslint-config-sentry-react/index.js

```diff
@@ -19,6 +19,20 @@ module.exports = {
     'getsentry/jsx-needs-il8n': ['off'],
     'testing-library/render-result-naming-convention': 'off',
     'testing-library/no-unnecessary-act': 'off',
+
+    // Disabled as we have many tests which render as simple validations
+    'jest/expect-expect': 'off',
```

Comment:
do you have an example?

Reply:
Here's one https://github.com/getsentry/sentry/blob/fab1c22ce843c70fe0119256ddfcad881299a5d2/tests/js/spec/components/searchSyntax/parser.spec.tsx#L64-L68

I would have to re-run it with this rule back on.

This is a new rule that got added, so I opted to just keep it off and we can decide if we want to turn it on later.

Reply:
oh sorry, wrong thing

Reply:
I'm talking about tests where all they do is just `renderWithTheme`.

Unfortunately there's a non-trivial amount of them